src/cljs/rems/collapsible.cljs

```diff
@@ -2,56 +2,82 @@
   (:require [rems.text :refer [text]]
             [rems.guide-util :refer [component-info example]]))
 
+(defn- hide-callback [id callback]
+  (fn [event]
+    (.preventDefault event)
+    (let [element (js/$ (str "#" id))]
+      (.collapse element "show")
+      (.focus element))
+    ;; bootstrap's .collapse returns immediately, so in order to avoid
+    ;; momentarily showing both buttons we wait for a hidden.bs.collapse event
+    (.. (js/$ (str "." id "-more"))
```

Comment:
I think this part looks nasty but I guess we have to live with it.

Looking forward to the demo!